scripts/common/common.sh

```diff
@@ -604,15 +604,21 @@ function kubeconfig_setup_outro() {
             cp "$(${K8S_DISTRO}_get_kubeconfig)" $ownerdir/.kube/config
             chown -R $owner $ownerdir/.kube
 
-            printf "To access the cluster with kubectl, ensure the KUBECONFIG environment variable is unset:\n"
+            printf "To access the cluster with kubectl:\n"
             printf "\n"
-            printf "${GREEN}    echo unset KUBECONFIG >> ~/.bash_profile${NC}\n"
             printf "${GREEN}    bash -l${NC}\n"
+            printf "Kurl uses "$(${K8S_DISTRO}_get_kubeconfig)", you might want to unset KUBECONFIG to use .kube/config:\n"
```

Comment:
This is IMO still missing the point that _we're the ones that are adding the `KUBECONFIG` env var to the bash profile_, and we should stop doing that

Reply:
Hi @laverya, @ricardomaraschini.

Following the comments inline 

> This is IMO still missing the point that we're the ones that are adding the KUBECONFIG env var to the bash profile, and we should stop doing that

kurl (kubeadmin) uses  `/etc/kubernetes/admin.conf` we never spoke in stop to do so or change that. 


> I agree. It is also not clear what we are trying to solve with this

What we are trying to do here is:
- Stop to say for the user that is required unset KUBECONFIG to have access to the kubectl because that is NOT required at all. 
- Make clear that kURL uses and updates   `/etc/kubernetes/admin.conf`  and NOT kube/config. 

>  IMHO we should keep the file under /etc/kubernetes/admin.conf and just inform that (as we have been doing)
Yes, it is what we are doing here., We are only clarifying it for the user. 

>  That is a file that shouldn't be moved anywhere as it provides administrative rights to the cluster.
We are not moving any file in this PR, so could you please clarify what do you mean with?

Reply:
Yeah, my comments do not belong here and I think the train has already departed so, never mind. My sole point was: we should not copy `/etc/kubernetes/admin.conf` to the user's home directory (it is a file providing administrative rights to the cluster and I prefer to have a single source of truth).

Reply:
Hi @ricardomaraschini,

> we should not copy /etc/kubernetes/admin.conf to the user's home directory

We created the .kube/config using it today and before. 
That is the kubeadmin recommendation (also it is outputted by) see:

<img width="902" alt="Screenshot 2023-01-23 at 09 55 20" src="https://user-images.githubusercontent.com/7708031/214011026-a4b75043-ae44-4f87-8b3b-dcdcd9a8aaa6.png">

Kubeadmin docs: https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#more-information

Therefore, change how it works never was discussed. This PR only change the output texted because it is not accurate (it is not required to unset KUBECONFIG to get the kubectl working at all) and clarifies that  `/etc/kubernetes/admin.conf` is what is used by kURL.

Reply:
Yes, thank you.

Reply:
@laverya, could you please check out the above comment and let us know if that is also making sense for you now and if we have your /lgtm /approval to move forward here? if not, could you please let us know why not?

Reply:
this is printing:
```
To access the cluster with kubectl:

    bash -l

Kurl uses "/etc/kubernetes/admin.conf", you might want to unset KUBECONFIG to use .kube/config:

    echo unset KUBECONFIG >> ~/.bash_profile
```

I'm still not sure this wording is ok.

Give me a minute to try and figure out something better

Reply:
Sure thx for the help.